lib/cli/commands/listswaps.ts

```diff
@@ -0,0 +1,93 @@
+import { callback, loadXudClient } from '../command';
+import { Arguments } from 'yargs';
+import { ListSwapsRequest, ListSwapsResponse } from '../../proto/xudrpc_pb';
+import Table , { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+
+const FAILED_HEADERS = [
+  colors.blue('peer public key'),
+  colors.blue('order id'),
+  colors.blue('trading pair'),
+  colors.blue('quantity'),
+  colors.blue('failure reason'),
+];
+// TODO: create a function that generates these tables
+const SUCCESS_HEADERS = [
```

Comment:
Do we want these to be capitalized like I think we're doing for other commands? Same for the failed table above.